q2_moshpit/plugin_setup.py

```diff
@@ -418,7 +418,8 @@
     function=q2_moshpit.dereplication.dereplicate_mags,
     inputs={
         "mags": SampleData[MAGs],
-        "distance_matrix": DistanceMatrix
+        "distance_matrix": DistanceMatrix,
+        "busco_results": BUSCOResults
```

Comment:
I think to be a bit more future-proof we should rather set this to be of the Metadata type and not only BUSCOResults. We would then need one more parameter which is the column name (set by default to "complete") and an additional check to make sure the column exists in the table. Before that, though, could you please check that the BUSCOResults -> q2.Metadata transformer exists? (I think it should but just in case)